docs/example_config.md

```diff
@@ -1,41 +1,53 @@
-```json
-{
-  "profiles": [
-    {
-      "name": "default",
-      "bugzilla-tracker-keyword": "tracker",
-      "bugzilla-url": "bugzilla.work.com",
-      "trello-board": "test board",
-      "entity-state-mapping": [
-        {
-          "bugzilla-states": [
-            "NEW"
-          ],
-          "trello-queue": "TODO"
-        },
-        {
-          "bugzilla-states": [
-            "ASSIGNED",
-            "MODIFIED",
-            "POST"
-          ],
-          "trello-queue": "IN-PROGRESS"
-        },
-        {
-          "bugzilla-states": [
-            "VERIFIED",
-            "CLOSED"
-          ],
-          "trello-queue": "DONE"
-        }
-      ],
-      "user-mapping": [
-        {
-          "bugzilla-user-name": "adarshvritant",
-          "trello-user-name": "vritantadarsh"
-        }
-      ]
-    }
-  ]
-}
+```yaml
+---
+-
+  profiles:
+    -
+      name: default
+      bugzilla:
+        tracker-keyword: tracker
+        url: bugzilla.work.com
+      rules:
+        - "rule 1"
+        - "rule 2"
+      trello:
+        api_key: aaa
+        api_secret: aab
+        oauth_token: aac
+        oauth_token_secret: aad
+    -
+      name: override
+      bugzilla:
+        tracker-keyword: tracker
+        url: bugzilla.home.com
+      rules:
+        - "rule 3"
+        - "rule 2"
+-
+  rules:
+    -
+      from: BUG_ASSIGNED
+      name: "rule 1"
+      to: CARD_BOARD_LIST
+    -
+      from: "BUG_ASSIGNED, BUG_NEW"
+      name: "rule 2"
+      to: CARD_BOARD_LIST
+    -
+      from: CARD_BOARD_LIST
+      name: "rule 3"
+      to: ISSUE_LABEL_NAME
+-
+  users:
+    -
+      bugzilla: mynick@bugzilla.com
+      github: mynick@github.com
+      nick: "my nick"
+      trello: mynick@mydomain.com
+    -
+      bugzilla: mynick@bugzilla.com
+      github: mynick@github.com
+      nick: "my nick 2"
+      trello: mynick@mydomain.com
+---
 ```
```

Comment:
Instead of:
```
rules:
    -
         name: "rule 1"
```

let's do:
```yaml
rules:
    -
        rule1:
                     from: BUG_ASSIGNED
                     to: CARD_BOARD_LIST
        rule2:
                     from: ....
                     to: ....
```

The processing of the rules would be easier if we do this since we can simply look up the corresponding rules key set in the profile from a rules dict.

```python
parsed_config = yaml.load(yaml_file)
rules = parsed_config['rules']

rule = rules[RULE_NAME_FROM_PROFILE]
```

Otherwise we'd have to iterate to match rule.name.